keyboards/trainpad/readme.md

```diff
@@ -0,0 +1,17 @@
+# trainpad
+
+Custom four key keyboard. **See each individual board for pin information.**
+
+* Keyboard Maintainer: [Ananya Kirti](https://github.com/AnanyaKirti)
+* Hardware Supported: custom stacked FR4 case
+* Hardware Availability: Contact bob at [IMKC Discord](https://discord.gg/imkc)
+
+Make example for this keyboard (after setting up your build environment):
+
+    make trainpad:default
+
+Flashing example for this keyboard:
+
+    make trainpad:default:flash
+
+See the [build environment setup](https://docs.qmk.fm/#/getting_started_build_tools) and the [make instructions](https://docs.qmk.fm/#/getting_started_make_guide) for more information. Brand new to QMK? Start with our [Complete Newbs Guide](https://docs.qmk.fm/#/newbs).
```

Comment:
Could you add instructions above this paragraph on how to enter bootloader mode for this keyboard?

Here's some sample text you can modify as needed for this purpose:

```
## Bootloader

Enter the bootloader in 3 ways:

* **Bootmagic reset**: Hold down the key at (0,0) in the matrix (usually the top left key or Escape) and plug in the keyboard
* **Physical reset button**: Briefly press the button on the back of the PCB - some may have pads you must short instead
* **Keycode in layout**: Press the key mapped to `QK_BOOT` if it is available
```

Reply:
done!